pkg/testutils/git.go

```diff
@@ -67,7 +64,7 @@ func CreateGitRemoteBranch(t *testing.T, dir string, branch string, remote strin
 
 // CreateSubdir TODO: comment
 func CreateSubdir(t *testing.T, dir string, subdir string) {
-	cmd := NewCommand(fmt.Sprintf("mkdir -p %s", subdir))
+	cmd := utils.NewCommand(fmt.Sprintf("mkdir -p %s", subdir))
```

Comment:
It is just creating dir. It might be better to use golang `os.MkDir` instead of execing